app-editors/vim/vim-9.0.0099.ebuild

```diff
@@ -0,0 +1,356 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+# Please bump with app-editors/vim-core and app-editors/gvim
+
+VIM_VERSION="9.0"
+LUA_COMPAT=( lua5-{1,3,4} luajit )
+PYTHON_COMPAT=( python3_{8..11} )
+PYTHON_REQ_USE="threads(+)"
+USE_RUBY="ruby26 ruby27 ruby30 ruby31"
```

Comment:
ruby26 is masked for removal so it can be removed here (it currently is a no-op). That also applies to the other commits where this was updated.

Reply:
OK, thanks!

Reply:
done